examples/lsm-nice/lsm-nice-ebpf/src/main.rs

```diff
@@ -27,22 +28,22 @@ pub fn task_setnice(ctx: LsmContext) -> i32 {
 
 // (3)
 unsafe fn try_task_setnice(ctx: LsmContext) -> Result<i32, i32> {
+
     let p: *const task_struct = ctx.arg(0);
     let nice: c_int = ctx.arg(1);
     let ret: c_int = ctx.arg(2);
 
-    // If previous eBPF LSM program didn't allow the action, return the
-    // previous error code.
+    info!(&ctx, "The PID supplied to this program is: {}, with nice value {} and return value {}", (*p).pid, nice, ret); 
+    info!(&ctx, "The PID that we need to monitor for changes: {}", core::ptr::read_volatile(&PID));
     if ret != 0 {
         return Err(ret);
     }
-
-    // Deny setting the nice value lower than 0 for the defined PID.
+    
     if (*p).pid == core::ptr::read_volatile(&PID) && nice < 0 {
+        info!(&ctx, "Nice value is less than zero, return error");
```

Comment:
should this log the value? also, it should say "returning" to avoid sounding like a directive

Reply:
I don't think it's necessary to log the nice value, since it's logged in the previous log message.